dask_expr/_collection.py

```diff
@@ -147,12 +147,16 @@ def __getitem__(self, other):
             return new_collection(self.expr.__getitem__(other.expr))
         return new_collection(self.expr.__getitem__(other))
 
-    def persist(self, fuse=True, combine_similar=True, **kwargs):
-        out = self.optimize(combine_similar=combine_similar, fuse=fuse)
+    def persist(self, simplify=True, fuse=True, combine_similar=True, **kwargs):
+        out = self.optimize(
+            lower=True, simplify=simplify, combine_similar=combine_similar, fuse=fuse
+        )
         return DaskMethodsMixin.persist(out, **kwargs)
 
-    def compute(self, fuse=True, combine_similar=True, **kwargs):
-        out = self.optimize(combine_similar=combine_similar, fuse=fuse)
+    def compute(self, simplify=True, fuse=True, combine_similar=True, **kwargs):
```

Comment:
I don’t want to add unnecessary keywords here and this will work differently after #294, so I wouldn’t add control over simplify here

Reply:
Yeah, the motivation isn't really to control options to `compute`, so I'm happy to roll that back. Thanks for pointing out 294.